accounts/models.py

```diff
@@ -350,55 +381,123 @@ def num_sounds_pending_moderation(self):
                 ~Q(sound__moderation_state='OK') &\
                 ~Q(status='closed')))
 
-    def get_info_before_delete_user(self, remove_sounds=False, remove_user=False):
+    def get_info_before_delete_user(self, include_sounds=False, include_other_related_objects=False):
```

Comment:
hah, I did not really invert the behaviour, it only looks like that because of the variable names. The method gets info about the things that will be removed. If you use `include_sounds`, it will list sounds as well. Before, using `remove_sounds` meant that the deletion action would remove sounds, so I want the info for the case of sounds being removed (so include sounds). I changed it to be clearer. There are 4 usages of that method and they use different settings for the parameters so parameters are needed.

Reply:
ok, that's great that you clarified it then, thanks